scripts/modules/CoverCalculator.js

```diff
@@ -8,6 +8,77 @@ import {queueUpdate} from "../../../simbuls-athenaeum/scripts/update-queue.js"
 
 const NAME = "CoverCalculator";
 
+export function getDemonlordTokenSizes() {
```

Comment:
Aren't these available in the System itself? Don't really want to start adding functions for token sizes etc specifc to systems. This is a slippery slope of maintenance.

Reply:
Unfortunatley there are no predefinied  [sizes categories ](https://github.com/Xacus/demonlord/blob/master/src/module/config.js) in demonlord system. Size is a [string](https://github.com/Xacus/demonlord/blob/master/src/template.json#L53) in demonlord.

![image](https://github.com/vtt-lair/simbuls-cover-calculator/assets/92884040/bac688d2-5e07-419d-9602-d9b6e06372ef)